packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -4,14 +4,14 @@ import { FieldExtensionSDK } from '@contentful/app-sdk';
 import { EntityProvider } from '@contentful/field-editor-reference';
 import { FieldConnector } from '@contentful/field-editor-shared';
 import * as Contentful from '@contentful/rich-text-types';
-// FIXME: extract this out to internal folder?
 import { Plate, getPlateActions } from '@udecode/plate-core';
```

Comment:
Intentionally didn't export this from `internal`?

Reply:
Yeah, decided that this makes sense as one place to actually use Plate as is. It felt fine to use here as it's the root of the project, what do you think?

Reply:
I don't think the Plate component will be used more than once